### DISCUSSION TITLE
Sms verification

### DISCUSSION THREAD

**Discussion by ozankahramanoglu** [python-telegram-bot/python-telegram-bot#2433] [Q&A] [answered]
Is there anyone that come intihar this issue?
It wont let me use my bot without giving away my sms verification code
Does anyone know how to disable this one
![Screenshot_20210315-210851_Telegram](https://user-images.githubusercontent.com/10221726/111200426-cb938080-85d2-11eb-8953-f0370dd70359.png)

**Comment by Bibo-Joshi** [comment, marked as answer]
This is the repository of `python-telegram-bot`, a python library to write telegram bots. If you have questions about a specific bot on Telegram, you should look for a support group or contact info of the maintainer of that bot.

**Comment by Welsyntoffie** [comment]
trolling
